kotlin/internal/jvm/BUILD

```diff
@@ -11,3 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+java_import(
+    name = "android_sdk",
+    neverlink = 1,
+    jars = ["@bazel_tools//tools/android:android_jar"],
```

Comment:
Surely there must be a better place to import this from ?

Reply:
@jin is this correct ?

Reply:
Yeah, that's the correct approach.

Reply:
(for now). This might change sometime in the future where it's moved into `@androidsdk`.